start.java

```diff
@@ -95,6 +102,17 @@ public static void main(String[] args) {
             commandLine.add("deployments/local/dev/docker-compose-openldap.yml");
         }
 
+        if (EXTENSIONS_OPT_CLASSES.equals(deployment)) {
+            commandLine.add("--file");
+            commandLine.add("deployments/local/dev/docker-compose-extensions-classes.yml");
+        } else if(EXTENSIONS_OPT_JAR.equals(deployment)) {
+            commandLine.add("--file");
+            commandLine.add("deployments/local/dev/docker-compose-extensions-jar.yml");
+        } else {
+            System.out.printf("Unkown deployment %s%n", deployment);
```

Comment:
Should we exit here?

Reply:
Good question, the text should contain a warning, that no extensions are mounted and just continue? I am also fine with exit.

Reply:
System.err.printf(...)

then I'd opt for System.exit(-1);

Reply:
:+1: